docs/manual/en/introduction/How-to-dispose-of-objects.html

```diff
@@ -54,11 +54,11 @@ <h2>Render Targets</h2>
 		for realizing custom rendering destinations. These objects are only deallocated by executing [page:WebGLRenderTarget.dispose]().
 	</p>
 
-	<h2>Scenes</h2>
+	<h2>Scenes and Cameras</h2>
 
 	<p>
-		The renderer maintains for scenes special data structures for sorting and rendering. If for some reasons a scene object becomes obsolete in an application,
-		call [page:Scene.dispose]() in order to free these resources.
+		The renderer maintains for scenes and cameras special data structures for sorting and rendering. If for some reasons a camera or a scene becomes obsolete in an
```

Comment:
>If for some reasons a camera or a scene becomes obsolete

What does that mean? Is there a better way to say this?

>The renderer maintains for scenes and cameras special data structures for sorting and rendering.

I do not think we need to explain what the data structures are for.

Reply:
> What does that mean? Is there a better way to say this?

TBH, I don't know. `Scene.dispose()` and `Camera.dispose()` are indented for the case you don't need a scene or camera object anymore. Is the term "obsolete" not right in this context?

Reply:
> I do not think we need to explain what the data structures are for.

A few users asked in the forum what's happening under the hood. So what is exactly cached and why. Since it's a guide and not a doc page, I thought adding this info is okay...

Reply:
>Is the term "obsolete" not right in this context?

I don't think so.

>A few users asked in the forum what's happening under the hood.

Then, tell them in the forum.

>So what is exactly cached and why.

What is written there is not descriptive enough to be useful.

The title is "How to dispose of objects". So you can say:

    If you delete a camera, or no longer need the camera for rendering, call camera.dispose() to clear camera-related data structures cached by WebGLRenderer.

Reply:
Okay, sounds good.